backend/app/email_service.py

```diff
@@ -78,22 +79,24 @@ def confirm_email(self, user, current_studies, tracking_code=None, logo_url=None
         token = ts.dumps(user.email, salt='email-reset-key')
         role = '' + user.role.name + '/'
 
+        ga_link = '?utm_source=email&utm_medium=referral&utm_campaign=reset_password&utm_content='\
```

Comment:
Same line many times, with a slight change, makes me think this should be a method somewhere that would help force consistency and make this more readable.